manager/eris-mint/evm/snative_test.go

```diff
@@ -0,0 +1,143 @@
+package vm
+
+import (
+	"encoding/hex"
+	"testing"
+
+	. "github.com/eris-ltd/eris-db/word256"
+	"github.com/stretchr/testify/assert"
+	"fmt"
+	. "github.com/eris-ltd/eris-db/manager/eris-mint/evm/opcodes"
+	ptypes "github.com/eris-ltd/eris-db/permission/types"
+)
+
+/* Compiling the Permissions solidity contract at
+(generated by Solidity() function)
+https://ethereum.github.io/browser-solidity yields:
+
+3fbf7da5 add_role(address,bytes32)
+744f5998 has_base(address,uint64)
+e8145855 has_role(address,bytes32)
+28fd0194 rm_role(address,bytes32)
+3f0ebb30 set_base(address,uint64,uint64)
+67dc6f70 set_global(address,uint64,uint64)
+73448c99 unset_base(address,uint64)
+*/
+
+func TestPermissionsContractSignatures(t *testing.T) {
+	contract := SNativeContracts()["permissions_contract"]
+
+	assertContractFunction(t, contract, "3fbf7da5",
+		"add_role(address,bytes32)")
```

Comment:
Ah right yeah, I can do that, but it will involve a few more changes. Obviously just a search and replace job in Eris db so I can do it, but it got me to question the wisdom, will do it if you and Jason and Ben think it's good idea